tests/test_library_conf.py

```diff
@@ -8,10 +8,6 @@
 
 
 @irrelevant(library="cpp")
```

Comment:
```suggestion
@irrelevant(library="cpp")
@released(dotnet="xx.xx.xx")
```

where `xx.xx.xx` is the good version number (tests are executed on previous versions)